mockery/generator_test.go

```diff
@@ -302,6 +302,18 @@ type RequesterPtr struct {
 	mock.Mock
 }
 
+type RequesterPtr_Get struct {
+	*mock.Call
+}
+
+func (_m *RequesterPtr_Get) Return (str *string, err error) {
+}
```

Comment:
Shouldn't this function be doing something? Like calling the `*mock.Call` object?